docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,264 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fast Fourier Transform (FFT)\n",
+    "The following gives background information that is helpful to understand how the Fast Fourier Transform (FFT) and the corresponding normalizations are defined in pyfar and how these are related to the concepts of energy and power signals."
```

Comment:
Maybe add a placeholder `LINK-TO-AUDIO-OBJETCS-NOTEBOOK` and mention this as a source for more information about energy and power signals.

Reply:
added link to "FFT normalizations" paragraph